packages/web/src/components/dynamic-image/DynamicImage.tsx

```diff
@@ -85,7 +85,7 @@ const fadeIn = (
       ? `url(${image})`
       : image
     ref.current.style.transition = `opacity ${
-      immediate ? '0.1s' : '0.5s'
+      immediate ? '0.1s' : '0.6s'
```

Comment:
I think I will pull this back to 0.3s or something before merging, but want to discuss next week

Reply:
staging is epicly fast with this + steve's changes